### PR TITLE
ARC health endpoint

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: downloadservice/app.py [hunk 1]
```diff
@@ -137,6 +140,28 @@ def get_upstream_session():
...
+
+
+@app.route(url_prefix + "/health/arc")
+def health_arc():
+    try:
+        return jsonify(arcinfo())
+    except Exception as e:
+        print(e)
+        return f"Unhealthy! {e}", 500
+
```
Comment (volodymyrss) [inline downloadservice/app.py:162]:
```suggestion
```

File: downloadservice/app.py [hunk 2]
```diff
@@ -286,7 +287,33 @@ def get_upstream_session(user=None):
...
+            re.search(r"Total slots: ([0-9]*)", arcinfo.decode()).group(1)),
+    )
+
+
+@app.route(url_prefix + "/health/arc")
+def health_arc():
+    try:
+        return jsonify(arcinfo())
+    except Exception as e:
+        logger.exception("arcinfo result is not good")
```
Comment (Ph0tonic) [inline downloadservice/app.py:312]:
```suggestion
        logger.exception(f"arcinfo result is not good: {e}")
```

**Review by Ph0tonic — APPROVED** [review]
LGTM, maybe we can display the error in the logs.

File: downloadservice/app.py (hunk 2 above)
Comment (volodymyrss) [reply downloadservice/app.py:312]: I set auto-merge so your approval dismissed your comment :(
But, in this case, exception is added anyway by the logger (https://docs.python.org/3/library/logging.html#logging.exception) :)

**Pull request merged**